index.html

```diff
@@ -4987,8 +4977,20 @@ <h2 id="AudioWorkletGlobalScope">
             This special execution context is designed to enable the
             generation, processing, and analysis of audio data directly using
             JavaScript in the audio <a>rendering thread</a>. The user-supplied
-            script code is evaluated in this scope to define an
-            <a>AudioWorkletProcessor</a>.
+            script code is evaluated in this scope to define one or more
+            <a>AudioWorkletProcessor</a> subclasses, which in turn are used to
+            instantiate <a>AudioWorkletProcessor</a>s, in a 1:1 association
+            with <a>AudioWorkletNode</a>s in the main scope.
+          </p>
+          <p>
+            At least one <a>AudioWorkletGlobalScope</a> exists for each
+            <a>AudioContext</a> that contains one or more
+            <a>AudioWorkletNode</a>s. The running of imported module scripts is
```

Comment:
Can we leave 'module' out? Or do you have a reference this is actually the 'module' pattern?